packages/client/components/ActivityLibrary/ActivityCard.tsx

```diff
@@ -72,6 +109,17 @@ export const ActivityCard = (props: ActivityCardProps) => {
           </div>
         </div>
       )}
+      {template && (
+        <Tooltip open={showTooltip}>
```

Comment:
Sometime position is a bit weird, this might be a separate issue with tooltip component itself or we may need to specify some radix props regarding position

<img width="509" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/1375c8c7-d719-4612-84da-7476022e6ea6">

Reply:
Yeah, we had some debate about where the tooltip should go. Maybe in a fast-follow PR I move it below the card at least?

Reply:
Initially we thought the tooltip could hang from and the mouse pointer:
![Captura de pantalla 2023-11-28 a la(s) 3 30 42 p m](https://github.com/ParabolInc/parabol/assets/3276087/74668e59-2800-499c-a033-f1690ede8eb9)